File: AnalyseData/src/main/java/fr/polytech/unice/creadirama/analyse/dto/SimulationPerDay.java
package fr.polytech.unice.creadirama.analyse.dto;

import fr.polytech.unice.creadirama.analyse.entity.FeeBtw2Day;
import fr.polytech.unice.creadirama.analyse.entity.Transaction;
import lombok.*;
import org.joda.time.DateTime;
import org.joda.time.format.DateTimeFormat;

import java.util.Map;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@EqualsAndHashCode
public class SimulationPerDay {

    private double sum;

    private double avg;

    /*private Transaction minTransaction;

    private Transaction maxTransaction;*/

    private int nbTransaction;

}

File: Flutter/credirama/lib/model/simulationPerDay.dart
class SimulationPerDay {

  double sum;
  double avg;
  int nbTransaction;

  SimulationPerDay({double sum, double avg, int nbTransaction}) {
    this.sum = sum;
    this.avg = avg;
    this.nbTransaction = nbTransaction;
  }

  factory SimulationPerDay.fromJson(Map<String, dynamic> json) {
    return SimulationPerDay(
        sum: json['sum'],
        avg: json['avg'],
        nbTransaction: json['nbTransaction']);
  }

  @override
  String toString() {
    return 'SimulationPerDay{sum: $sum, avg: $avg, nbTransaction: $nbTransaction}';
  }

}
